vm_supervisor/snapshots.py

```diff
@@ -0,0 +1,112 @@
+import logging
+from pathlib import Path
+from typing import Dict, Optional
+
+from aleph_message.models import ItemHash
+from schedule import Job, Scheduler
+
+from .conf import SnapshotCompressionAlgorithm, settings
+from .models import VmExecution
+from .storage import compress_volume_snapshot, create_volume_snapshot
+
+logger = logging.getLogger(__name__)
+
+
+class DiskVolumeFile:
+    path: Path
+    size_mib: int
```

Comment:
Why don't you just store the size in bytes? I don't see the advantage of converting to MiB.